packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -178,6 +182,36 @@ export class Span implements api.Span, ReadableSpan {
     return true;
   }
 
+  recordException(exception: api.Exception, time: api.TimeInput = hrTime()) {
+    const attributes: api.Attributes = {};
+    if (!exception) {
```

Comment:
No need for this check. It will just fall down to the error log.

Reply:
if exception is undefined null etc. this check will make it will not create a null pointer exception - there is a test for that

Reply:
@obecny is right, we still need to take care of misuse in JS

Reply:
@vmarchaud if `null` or `undefined` is passed into `recordException`, it works fine without the check. See the current version which does not have this check.